src/System.Collections.Immutable/src/System/Collections/Immutable/ImmutableEnumerable.DebuggerProxy.cs

```diff
@@ -0,0 +1,47 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+
+namespace System.Collections.Immutable
+{
+    /// <summary>
+    /// Displays immutable enumerables in the debugger.
+    /// </summary>
+    /// <typeparam name="T">The element type of the enumerable.</typeparam>
+    /// <remarks>
+    /// This class should only be used with immutable enumerables, since it
+    /// caches the enumerable into an array for display in the debugger.
+    /// </remarks>
+    internal sealed class ImmutableEnumerableDebuggerProxy<T>
```

Comment:
The extra period convention is used for "sub-types" (for lack of a better term), e.g. the `ImmutableQueueDebuggerProxy` type was in the `ImmutableQueue_1.DebuggerProxy.cs` file because it was a "sub-type" of `ImmutableQueue<T>`.

Now that there's a single DebuggerProxy type for all collections, I don't think it makes sense to use the extra period convention as there is no `ImmutableEnumerable` "parent-type".

This should be like the other one-off helper types under /src/System/Collections/Immutable that apply to all of the immutable collections, like RefAsValueType.cs, e.g. DebuggerProxy.cs.